packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -561,25 +581,19 @@ describe('Transfer tests', function(this: any) {
         })
 
         describe('Transfer CeloDollars', () => {
-          describe('gasCurrency = CeloDollars >', () => {
+          describe('feeCurrency = CeloDollars >', () => {
             testTransferToken({
               expectedGas: 175303,
               transferToken: CeloContract.StableToken,
               feeToken: CeloContract.StableToken,
-              txOptions: {
-                gasFeeRecipient: FeeRecipientAddress,
```

Comment:
It's intentional. My main motivation for this is that when we are in the `full` sync mode we want the gateway fee fields to be empty. I decided to delete that field because it allows the test to cover that the defaults should be assigned appropriately by the `light` and `ultralight` nodes.